### PR TITLE
codeFormatting

### PR DESCRIPTION
**Pull request by MunifNagi**
- code formatting
-  proper naming for methods and variables
- control flow of the application

### CODE REVIEW

File: src/classes/AirPlane.java [hunk 1]
```diff
@@ -5,34 +5,30 @@
...
     private int bookedSeats;
     private char[][] seats;
 
-    public AirPlane(String planeName, String model, int rows) {
+    public AirPlane(String planeName, String model, int rows, int cols) {
         this.name = planeName;
         this.modelNumber = model;
         this.rows = rows;
-        this.maxSeats = 10 * rows;
+        this.cols=cols;
```
Comment (VbKoshak) [inline src/classes/AirPlane.java]: this.cols = cols;

File: src/classes/AirPlane.java [hunk 2]
```diff
@@ -5,34 +5,30 @@
...
-            seats[i][3] = 'D';
-            seats[i][4] = 'E';
-            seats[i][5] = 'F';
-            seats[i][6] = 'G';
-            seats[i][7] = 'H';
-            seats[i][8] = 'I';
-            seats[i][9] = 'J';
+        this.seats = new char[this.rows][this.cols];
+        int firstCol = (int) 'A';
+        System.out.println(firstCol);
```
Comment (VbKoshak) [inline src/classes/AirPlane.java]: Do we need this?

File: src/classes/AirPlane.java [hunk 3]
```diff
@@ -42,15 +38,14 @@ public char[][] getSeats() {
     public void printSeats() {
         System.out.println("Row");
         for (int i = 0; i < this.seats.length; i++) {
-            System.out
-                    .println((i + 1) + "  " + seats[i][0] + " " + seats[i][1] + " " + seats[i][2] + "   " +
-                            seats[i][3] + " " + seats[i][4] + " " + seats[i][5] + " " + seats[i][6] + "   " +
-                            seats[i][7] + " " + seats[i][8] + " " + seats[i][9]);
-
+            System.out.print((i + 1)+ "  ");
+            for (int c=0; c<this.cols;c++){
```
Comment (VbKoshak) [inline src/classes/AirPlane.java]: for(int c = 0; c < this.cols; c++) {

File: src/classes/AirPlane.java [hunk 4]
```diff
@@ -59,8 +54,15 @@ public String getName() {
     }
 
     public int getCapacity() {
-        return this.maxSeats;
+        return this.capacity;
     }
+    public int getRows() {
```
Comment (VbKoshak) [inline src/classes/AirPlane.java:59]: empty line between methods

File: src/classes/Airlines.java [hunk 5]
```diff
@@ -14,21 +14,19 @@ public Airlines(String Airlines) {
...
-    public void getAllFlights() {
+    public void printAllFlights() {
         System.out.println("Airlines flights");
         for (int i = 0; i < flights.size(); i++) {
             System.out.println(flights.get(i) + " ");
         }
     }
-
-    public void getAllAirPlanes() {
+    public void printAllAirPlanes() {
```
Comment (VbKoshak) [inline src/classes/Airlines.java:30]: empty line between methods

File: src/classes/Airlines.java [hunk 6]
```diff
@@ -38,7 +36,6 @@ public void getAllAirPlanes() {
     public void addFlight(Flight flight, ArrayList<Flight> flights) {
         flights.add(flight);
     }
-
```
Comment (VbKoshak) [inline src/classes/Airlines.java]: return this line

File: src/classes/Main.java [hunk 7]
```diff
@@ -7,9 +7,9 @@ public static void main(String[] args) {
         AirPort JFK = new AirPort("JFK", "New York");
         AirPort LHR = new AirPort("LHR", "London");
 
-        AirPlane Boeing = new AirPlane("Boeing", "1234", 7);
-        AirPlane Boeing2 = new AirPlane("Boeing", "1235", 4);
-        AirPlane Boeing3 = new AirPlane("Boeing", "1236", 5);
+        AirPlane Boeing = new AirPlane("Boeing", "1234", 7,4);
```
Comment (VbKoshak) [inline src/classes/Main.java]: add a space between params:
AirPlane Boeing = new AirPlane("Boeing", "1234", 7,` `4);

**Review by VbKoshak — CHANGES_REQUESTED** [review]
Just a code style

File: src/classes/AirPlane.java (hunk 2 above)
Comment (MunifNagi) [reply src/classes/AirPlane.java]: I forgot to remove the print statement but the firstCol one needed that value inside the for loop
`seats[r][c]= (char) ( firstCol + c);`

**Review by VbKoshak — APPROVED** [review]
lgtm

**Comment by MunifNagi** [comment]
Outdated

**Pull request closed without merging**